Gruntfile.js

```diff
@@ -153,6 +153,7 @@ module.exports = (grunt) => {
       },
       mocha: {
         options: {
+          esversion: 8,
```

Comment:
async / await

Reply:
Why does JSHint have `esversion: 6`?
Does it not support anything higher?

Reply:
Because that is what `esnext` was before https://jshint.com/docs/options/#esnext.

Reply:
Makes sense, but can we set it to `esversion: 8` for the test files (and node files)?

Reply:
Yes, we will but not as part of this PR.